ros2_ws/src/otto_fleet_nav/launch/otto_nav_launch.py

```diff
@@ -156,7 +156,7 @@ def generate_launch_description():
 
     ld.add_action(rviz_cmd)
     ld.add_action(bringup_cmd)
-    ld.add_action(spawner)
+    # ld.add_action(spawner)
```

Comment:
Is this change needed? How does it affect the demo in a small scene?

Reply:
This line is not needed and should be removed.